### PR TITLE
QAE-217 - Test case for Primary menu submenu border color in the customizer

### PR DESCRIPTION
**Pull request by bsfprajaktap**
Description
This test case is for adding border color to the submenu

Screenshots
https://share.bsf.io/jku4XdB7

Types of changes
Test case for adding border color to submenu

How has this been tested?

1. Create pages.
2. Create menu and sub menu
3. Add border color to the sub menu in customizer settings.

cmd to execute TC-
npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color.test.js
Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { createNewMenu } from '../../../../utils/create-menu';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Add sub menu for primary menu and add border to the sub menu', () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js:4]: description should be test case specific- i.e color

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	it( 'sub menu should be added successfully', async () => {
+		await createNewMenu();
+		//center alignment for desktop, tablet and mobile
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js:7]: remove unwanted comment

File: tests/e2e/utils/create-menu.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/utils/create-menu.js:1]: check "publish-post" util path

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		} );
+		await page.hover( '.menu-link' );
+		await expect( {
+			selector: '.sub-menu',
+			property: 'border-color',
+		} ).cssValueToBe(
+			`${ submenuBorder[ 'header-menu1-submenu-b-color' ] }`,
+		);
+	} );
+} );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js:46]: rename the test case as "border-color"

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+		};
+		await setCustomize( headerMenuAlignment );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '#primary-site-navigation' );
+		await expect( true ).toBe( true );
+	} );
+	it( 'border color to the submenu should be added correctly', async () => {
+		const submenuBorder = {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/primary-menu-submenu-border-color.test.js:25]: constant should be as "submenuBorderColor"

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color.test.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+				right: 10,
+			},
+		};
+		await setCustomize( submenuBorderColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.hover( '.menu-link' );
+		await expect( {
+			selector: '.sub-menu',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color.test.js:39]: Please check selector:- https://share.bsf.io/7KuQbv06

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/header-builder/primary-menu-submenu/border-color.test.js:40]: modified selector

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**